collator/src/lib.rs

```diff
@@ -359,7 +359,7 @@ fn build_collator_service<SP, P, C, R, Extrinsic>(
 
 			let future = silenced.map(drop);
 
-			tokio::spawn(future);
+			spawner.spawn_obj(Box::new(future).into()).expect("could not run build collator task");
```

Comment:
If you import [`SpawnExt`](https://docs.rs/futures/0.3.5/futures/task/trait.SpawnExt.html) you can do this instead:
```suggestion
			spawner.spawn(future).expect("could not run build collator task");
```

Reply:
It doesn't work as spawner only impl Spawn: https://docs.rs/futures/0.3.5/futures/task/trait.Spawn.html

Reply:
https://docs.rs/futures/0.3.5/futures/task/trait.SpawnExt.html#implementors I thought it was implemented for all `Spawn` things?

Reply:
You are right! I just needed the trait in scope :facepalm: